### PR TITLE
Update Direct Deposit to match new standards

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://design.va.gov/patterns/form-labels has been updated with new labels 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/13965 is for updating the 1995 form, 
since the labels and content comes from platform common code and edu benefits common code respectively this PR is to update the labels and content for all forms using this page

## Screenshots
Screen Shot 2020-10-05 at 11 33 15 AM
77c273261b4a.png)

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/platform/forms/definitions/bankAccount.js [hunk 1]
```diff
@@ -25,10 +25,10 @@ const uiSchema = {
     },
   },
   accountNumber: {
-    'ui:title': 'Account number',
+    'ui:title': 'Bank account number',
```
Comment (rianfowler) [inline src/platform/forms/definitions/bankAccount.js:28]: edu forms impacted by this: 
- 1995
- 5495
- 0994
- 10203

File: src/applications/edu-benefits/pages/directDepositChange.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ export default function createDirectDepositChangePage(schema) {
       bankAccount: _.merge(bankAccountUI, {
         'ui:options': {
           hideIf: formData => !isStartUpdate(formData),
+          expandUnder: 'bankAccountChange',
```
Comment (rianfowler) [inline src/applications/edu-benefits/pages/directDepositChange.js:29]: Edu forms impacted by this:
- 1995
- 5495

**Review by rianfowler — APPROVED** [review]
I don't see codeowners set for most of the forms this change impacts but the changes seem fine from a technical perspective.

**Review by bkjohnson — COMMENTED** [review]
Will you also update this file, in case an app decides to import from it before we combine these files into one?

https://github.com/department-of-veterans-affairs/vets-website/blob/81f9ce9c34986e04960c146b84e4fff4df3c8ce5/src/platform/forms-system/src/js/definitions/bankAccount.js#L19

**Comment by zurbergram** [comment]
@bkjohnson it appears that https://github.com/department-of-veterans-affairs/vets-website/blob/81f9ce9c34986e04960c146b84e4fff4df3c8ce5/src/platform/forms-system/src/js/definitions/bankAccount.js#L19 is not used
Screen Shot 2020-10-06 at 4 09 25 PM

**Comment by bkjohnson** [comment]
Yes, I am aware.

> Will you also update this file, **in case an app decides to import from it before we combine these files into one?**

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**